excludes.txt

```diff
@@ -0,0 +1,26 @@
+fmt.Fprintln
```

Comment:
Did you mean to check in this file? Is it supposed to remain in sync with .glolangci.yaml? Do we have a process for doing this? If this was only for your local use, please remove it.

Reply:
Whoops, no that was just for helping me test the `errcheck` linter specifically.